programs/locking/src/lib.rs

```diff
@@ -196,27 +158,19 @@ mod locking {
                 },
                 signer,
             ),
-            unmint_amount,
+            unlock_amount,
         )?;
 
-        emit!(EventUnStake {
-            unmint_amount,
-            token_amount: withdraw_amount,
+        emit!(EventUnlock {
+            unmint_amount: unlock_amount,
+            token_amount: unlock_amount,
```

Comment:
they both the same, so we just need 1 field

Reply:
Fixed